.github/ISSUE_TEMPLATE/10-issue_report.yaml

```diff
@@ -0,0 +1,136 @@
+name: Report a Bug
+title: "<title>"
+description: |
+  You found a bug in Bareos causing the application to crash or
+  throw an exception, a component is buggy, or something looks wrong.
+labels: needs triage
```

Comment:
maybe it's intentional, but don't we want to also add the "bug" label here?

Reply:
As discussed we will auto-add the `bug` label, which will be easier to remove during triage.